.gitignore

```diff
@@ -90,6 +90,7 @@ src/media/profile_images/*
 src/urls\.txt
 
 src/files/styling/
+src/files/bepress/
```

Comment:
Could use a comment here as this is out of context without the plugin

Reply:
Maybe we should have a folder for plugins to store their files in and ignore that?

Reply:
Sounds good to me, `src/files/plugins/<plugin_name>` ?

Reply:
Exactamundo! I'll make the change to this and the bepress plugin.